test/python/test_mapper.py

```diff
@@ -52,10 +52,13 @@ def test_math_domain_error(self):
         """
         self.qp.load_qasm_file(self._get_resource_path('qasm/math_domain_error.qasm'), name='test')
         coupling_map = [[0, 2], [1, 2], [2, 3]]
-        result1 = self.qp.execute(["test"], backend="local_qasm_simulator",
-                                  coupling_map=coupling_map, seed=self.seed)
-
-        self.assertEqual(result1.get_counts("test"), {'0001': 480, '0101': 544})
+        result = self.qp.execute("test", backend="local_qasm_simulator",
+                                 coupling_map=coupling_map,
+                                 seed=self.seed, shots=1024)
+        counts = result.get_counts("test")
+        target = {'0001': 480, '0101': 544}
```

Comment:
I think for this circuit the ideal counts should be `shots / 2`. Can you change this block to:
```python
shots = 1024
result = self.qp.execute("test", backend="local_qasm_simulator",
                                 coupling_map=coupling_map,
                                 seed=self.seed, shots=shots)
counts = result.get_counts("test")
target = {'0001': shots / 2, '0101':  shots / 2}
threshold = 0.025 * shots
```
and check if the test still passes (you might need to increase the shots or adjust the threshold a little bit)